src/Orchard.Web/Modules/Orchard.ImportExport/Services/SetupService.cs

```diff
@@ -101,6 +102,10 @@ public class SetupService : Component, ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
```

Comment:
I am becoming less and less fond of this duplicated `SetupService`. :( Why is that necessary again??

In any case, since we allow the other schema builder operations directly in this code, I think we should either do the same with the distributed lock record, or refactor **all** of them into some kind of `CoreSchemaBuilder` responsible for creating all these low level tables.